File: sample/src/main/java/com/lynx/wind/recyclersample/view/MainActivity.java
package com.lynx.wind.recyclersample.view;

import android.content.Intent;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;
import android.view.View;

import com.lynx.wind.recyclersample.R;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        findViewById(R.id.btn_simple).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(MainActivity.this, SimpleActivity.class));
            }
        });

        findViewById(R.id.btn_filter).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(MainActivity.this, FilterActivity.class));
            }
        });

        findViewById(R.id.btn_load).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(MainActivity.this, PaginationActivity.class));
            }
        });
    }
}

File: sample/src/main/java/com/lynx/wind/recyclersample/view/MainActivity.kt
package com.lynx.wind.recyclersample.view

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import com.lynx.wind.recyclersample.R
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        btn_simple.setOnClickListener {
            startActivity(Intent(this@MainActivity, SimpleActivity::class.java))
        }

        btn_filter.setOnClickListener {
            startActivity(Intent(this@MainActivity, FilterActivity::class.java))
        }

        btn_load.setOnClickListener {
            startActivity(Intent(this@MainActivity, LoadMoreActivity::class.java))
        }
    }
}
